### PR TITLE
[PS-250] Prevent timeouts less than 1 minute

### PR DESCRIPTION
**Pull request by gbubemismith**
## Type of change

- [X] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective
Web Vault should not allow the user to select a value less than 1 minute in the custom timeout inputs.

## Code changes
 - **src/app/settings/preferences.component.ts:** a check to know if minutes entered is valid happens in the `submit` method.
 - **src/app/settings/vault-timeout-input.component.html:** minimum timeout error message display 
 - **src/locales/en/messages.json:** error text message
 - **src/app/modules/vault/modules/individual-vault/individual-vault.component.ts:** removed unused reference

PR is related to https://github.com/bitwarden/jslib/pull/802

https://user-images.githubusercontent.com/13024008/170232966-482a757e-791f-48c2-9469-ff8c2032ce21.mov

## Before you submit

- [X] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: src/app/settings/vault-timeout-input.component.ts
Comment (eliykat) [inline src/app/settings/vault-timeout-input.component.ts:24]: This method repeats the validation check you did in your jslib PR. Suggested solution here: https://github.com/bitwarden/jslib/pull/802#discussion_r882324921

File: src/app/settings/vault-timeout-input.component.ts
Comment (eliykat) [inline src/app/settings/vault-timeout-input.component.ts:23]: No need to add comments unless the code itself is unclear - this is a pretty simple method, so the method name should be clear enough to speak for itself.

File: src/app/settings/vault-timeout-input.component.html
Comment (eliykat) [inline src/app/settings/vault-timeout-input.component.html:50]: What is this icon? It doesn't show up for me.

File: src/app/settings/preferences.component.ts
Comment (eliykat) [inline src/app/settings/preferences.component.ts:85]: I know Danielle asked for this, but I don't want a pattern where we have to handle individual errors at the form level. It just doesn't scale, we'll end up with a million `if` statements. Can you check with her whether we can just change this to a generic message, like "The form has errors, please review and try again"? (or whatever wording she suggests)

**Review by eliykat — CHANGES_REQUESTED** [review]

File: src/app/settings/vault-timeout-input.component.html
Comment (gbubemismith) [reply src/app/settings/vault-timeout-input.component.html:50]: That's an icon I used for testing and forgot to remove

**Review by eliykat — COMMENTED** [review]
LGTM once your jslib changes have been merged and you update the name of `getValidMinutesInput()` to match. :D

**Comment by eliykat** [comment]
I've approved your jslib pr. However, the web client just moved to the monorepo (`clients`), so you'll need to move these changes + this PR over there.

**Pull request closed without merging**
